tools/benchmarks/ml/mlperf/scripts/bootstrap-mlperf.sh

```diff
@@ -46,10 +49,22 @@ function install_tools() {
 }
 
 function install_libaries() {
+    pip -qq install tensorboardX h5py
+}
 
+function install_mlperf_tools() {
+    pip -qq install "git+https://github.com/mlcommons/logging.git@2.0.0"
 }
 
 function install_mlperf() {
+  mkdir -p $MLPERF_TMP
+  cd $MLPERF_TMP
+  if [ ! -d "cloudtik" ]; then
+    git clone https://github.com/oap-project/cloudtik.git
+  fi
```

Comment:
suggest to remove cloudtik folder if it exists and then always git clone.
You don't know what's the status of cloudtik folder even if it exists